.github/workflows/flaky-tests.yml

```diff
@@ -12,20 +12,26 @@ jobs:
         runs-on: ubuntu-latest
         if: ${{ github.event.workflow_run.conclusion == 'success' }}
         steps:
+            # Checkout defaults to using the branch which triggered the event, which
+            # isn't necessarily `trunk` (e.g. in the case of a merge).
             - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+              with:
+                  ref: trunk
 
             - name: Use desired version of NodeJS
               uses: actions/setup-node@5b52f097d36d4b0b2f94ed6de710023fbb8b2236 # v3.1.0
               with:
                   node-version-file: '.nvmrc'
                   cache: npm
 
-            - name: Npm install
+            - name: Npm install and build
+              # We don't have to build the entire project, just the action itself.
               run: |
                   npm ci
+                  npm run build:package-types
 
             - name: Report flaky tests
-              uses: ./.github/report-flaky-tests
+              uses: ./packages/report-flaky-tests
```

Comment:
`report-flaky-tests` doesn't have an `index.js`. Is that right? How does GitHub Actions know to run `build/index.js`?

Reply:
I believe GH Action will look for the entry in [`action.yml`](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#creating-an-action-metadata-file), which points to `build/index.js`.